test/e2e/connectivity_test.go

```diff
@@ -192,7 +192,12 @@ func TestPodConnectivityDifferentNodes(t *testing.T) {
 
 func (data *TestData) redeployAntrea(t *testing.T, enableIPSec bool) {
 	var err error
-
+	// export logs before deleting Antrea
+	if enableIPSec {
+		exportLogs(t, data, "beforeEnableIPSec", false)
+	} else {
+		exportLogs(t, data, "beforeDisableIPSec", false)
```

Comment:
redeployAntrea is used by `TestPodConnectivityAfterAntreaRestart` as well and might be used for other tests later. It might be a little confusing that other tests has a log directory named "beforeDisableIPSec" while it never enables it, how about "beforeRedeployAntreaWithIPSecEnabled" or "preRedeployAntreaWithIPSecEnabled"?

Reply:
+1, that's a good point

maybe just "beforeRedploy" and "beforeRedeployWithIPsecEnabled"

Reply:
@tnqn @antoninbas Sounds good, i'll update it.

Reply:
updated